frontend/src/modules/lobby/view/Lobby.vue

```diff
@@ -3,18 +3,24 @@
     <Header title="Welcome to Not Scribble" />
     <Header class="lobby-header-room-id" :title="lobbyId" />
     <div class="lobby-buttons">
-      <b-button class="lobby-button" variant="success" v-on:click="goToScribble">Start game</b-button>
+      <b-button class="lobby-button" variant="success" v-on:click="goToScribble"
+        >Start game</b-button
+      >
       <Nickname @onChangeNickname="onChangeNickname" />
-      <b-button class="exit-button" variant="danger" v-on:click="exitToHome">Exit Lobby</b-button>
+      <b-button class="exit-button" variant="danger" v-on:click="exitToHome"
+        >Exit Lobby</b-button
+      >
```

Comment:
This formatting is quite strange. I would recommend doing it like this. 
```
<b-button class="exit-button" variant="danger" v-on:click="exitToHome">
	Exit Lobby
</b-button>
```

Goes for everything in this pull request.